client/evilspammer/spamming_functions.go

```diff
@@ -51,11 +41,33 @@ func CustomConflictSpammingFunc(s *Spammer) {
 			wg.Add(1)
 			go func(clt evilwallet.Client, tx *ledgerstate.Transaction) {
 				defer wg.Done()
-				clt.PostTransaction(tx)
+				allSolid := handleSolidityForReuseOutputs(clt, tx, s)
+				if !allSolid {
+					s.ErrCounter.CountError(errors.Errorf("not all inputs are solid, txID: %s", tx.ID().Base58()))
+					return
+				}
+				ok := s.PostTransaction(tx, clt)
+				if ok {
+					if s.EvilScenario.OutputWallet.Type() == evilwallet.Reuse {
+						s.EvilWallet.SetTxOutputsSolid(tx.Essence().Outputs(), clt.Url())
+					}
+				}
```

Comment:
Maybe the logic here is something we should hide in the `PostTransaction`, including the solidity check?